CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+### 1.1.5
+* Add :raw serializer
+
+### 1.1.4
+* Add custom formatter feature
+* Tweak Travis CI tasks
+
+### 1.1.3
+* Send query_string to endpoint_url
+
+### 1.1.2
```

Comment:
Please use `##` on this row and above, so that the version number is consistently using the second heading. See below for more examples of the correct usage. Bonus points if you could also add the date-stamps for when these versions were released by checking against the git log.